app/views/mailing_list/steps/completed/shared/_is_teaching_right_for_me.html.erb

```diff
@@ -0,0 +1,4 @@
+<p>
+  Find out if <a href="/is-teaching-right-for-me">teaching is right for you</a>,
```

Comment:
```suggestion
  Find out <a href="/is-teaching-right-for-me">if teaching is right for you</a>,
```
@philspray Would it be better to put the `if` in the link text - link sounds like a bit of a statement rather than a question at the moment?